xyzspaces/spaces.py

```diff
@@ -1228,3 +1211,25 @@ def add_features_geobuf(
             features_size=features_size,
             chunk_size=chunk_size,
         )
+
+    def add_features_geopandas(
+        self, data, features_size: int = 2000, chunk_size: int = 1
+    ):
+        """
+        To upload data from Geopandas Dataframe file to a space.
+
+        :param data: Geopandas Dataframe to be uploaded
+        :param features_size: An int representing a number of features to upload at
```

Comment:
Skip the "int" as its visible in the signature, hence "The number of features..."